### ISSUE TITLE
Validator warning on valid CSP header values

### ISSUE THREAD

**Issue by duncan-bayne** [validator/validator#1943]
Hi, first up, thanks for vnu ❤️  I've been using it via the W3C validator pages for years, to check my markup on a number of projects.  However I've discovered an issue with false positives on pages hosted by https://pages.sr.ht/.

URL being validated or code to reproduce error:

https://validator.w3.org/nu/?doc=https%3A%2F%2Fhalp.fyi%2F (vnu 25.11.4).

```
Warning: Content-Security-Policy HTTP header: Bad content security policy: The sandbox directive should contain only allow -forms, allow -modals, allow -pointer-lock, allow -popups, allow -popups-to-escape-sandbox, allow -same-origin, allow -scripts, or allow -top-navigation. The sandbox directive should contain only allow -forms, allow -modals, allow -pointer-lock, allow -popups, allow -popups-to-escape-sandbox, allow -same-origin, allow -scripts, or allow -top-navigation.
```

It looks as though `allow-downloads` and `allow-presentation` are now valid values for the sandbox directive, per:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Content-Security-Policy/sandbox

... but vnu is still producing a warning.

**Comment by sideshowbarker** [comment]
Thanks for catching this and reporting it. I’ve fixed it, and deployed the fix:

https://validator.w3.org/nu/?doc=https%3A%2F%2Fhalp.fyi%2F
